website/templates/emails/conference_submitted.txt.mako

```diff
@@ -3,27 +3,26 @@ Hello ${fullname},
 Congratulations! You have successfully added your ${conf_full_name} ${presentation_type} to the Open Science Framework (OSF).
 
 % if user_created:
-Your account on the Open Science Framework has been created. To claim your account, please create a password by clicking here: [ ${set_password_url} ]. Please verify your profile information at [ ${profile_url} ].
+Your account on the Open Science Framework has been created. To claim your account, please create a password by clicking here: ${set_password_url}. Please verify your profile information at: ${profile_url}.
 
 % endif
-Your ${conf_full_name} submission has been added to the Open Science Framework. You now have a permanent, citable URL, that you can share: [ ${node_url} ]. All submissions ${conf_full_name} may be viewed at the following link: [ ${conf_view_url} ].
+You now have a permanent, citable URL, that you can share: ${node_url}. All submissions for ${conf_full_name} may be viewed at the following link: ${conf_view_url}.
```

Comment:
The sentence is different from the ticket description requirements. Could you verify that's what we want?